modules/civiremote_entity/civiremote_entity.routing.yml

```diff
@@ -0,0 +1,16 @@
+civiremote_entity.remote_page_get:
+  path: '/civiremote/get/{identifier}/{filename}'
+  defaults:
+    _controller: 'Drupal\civiremote_entity\Controller\CiviCRMPageController::get'
+    # The parameter filename is optional and is just used to show a convenient
+    # URL to the user.
+    filename: ~
+  options:
+    no_cache: TRUE
+    parameters:
+      identifier:
+        type: string
+      filename:
+        type: string
+  requirements:
+    _user_is_logged_in: 'TRUE'
```

Comment:
Not sure requiring a CiviRemote ID is actually necessary.

Reply:
You're right. The route is now unrestricted.